R/html-tweak.R

```diff
@@ -129,6 +134,133 @@ tweak_footnotes <- function(html) {
   xml2::xml_remove(container)
 }
 
+# Tabsets tweaking: find Markdown recommended in https://bookdown.org/yihui/rmarkdown-cookbook/html-tabs.html
+# and https://bookdown.org/yihui/rmarkdown/html-document.html#tabbed-sections
+# i.e. "## Heading {.tabset}" or "## Heading {.tabset .tabset-pills}"
+#  no matter the heading level -- the headings one level down are the tabs
+# and transform to tabsets HTML a la Bootstrap
+
+tweak_tabsets <- function(html) {
```

Comment:
I wonder if it's time to start breaking this file up into smaller pieces?

Reply:
Oh no, it was my favorite R script in pkgdown :cry: More seriously: https://github.com/r-lib/pkgdown/issues/1725